app/src/androidTest/java/com/github/sdpteam15/polyevents/view/activity/ItemRequestActivityTest.kt

```diff
@@ -216,26 +216,3 @@ class ItemRequestActivityTest {
 }
 
 
-// Source : https://stackoverflow.com/questions/28390574/checking-toast-message-in-android-espresso
-class ToastMatcher : TypeSafeMatcher<Root?>() {
-
-    private var currentFailures: Int = 0
-
-    override fun describeTo(description: Description) {
-        description.appendText("no toast found after")
-    }
-
-    override fun matchesSafely(item: Root?): Boolean {
-        val type: Int? = item?.windowLayoutParams?.get()?.type
-
-        if (TYPE_TOAST == type || TYPE_APPLICATION_OVERLAY == type) {
-            val windowToken = item.decorView.windowToken
-            val appToken = item.decorView.applicationWindowToken
-
-            if (windowToken == appToken) {
-                return true
-            }
-        }
-        return ++currentFailures >= 5
-    }
-}
```

Comment:
Why did you remove this test?

Reply:
I removed the tests because (I believe) @cpittet tried to test if the toast were shown or not. I believe that this doesn't work and we never use it (Android Studio tells me to remove it since this class was not used in the code) :)